src/components/Widget.tsx

```diff
@@ -97,6 +97,7 @@ export interface WidgetProps {
   tokenList?: string | TokenInfo[]
   width?: string | number
   dialog?: HTMLDivElement | null
+  disableBranding?: boolean
```

Comment:
As a sync'ed setting, instead of adding this to widget props, extend widget props from this: `WidgetProps extends BrandingSettings`. Should you add any settings to the `BrandingSettings` type, you won't need to touch the `Widget.tsx` file again.